packages/campaignclassic/README.md

```diff
@@ -0,0 +1,170 @@
+```
+
+## Usage
+
+### Initializing and registering the extension
+
+Initialization of the SDK should be done in native code, documentation on how to initialize the SDK can be found [here](https://github.com/adobe/aepsdk-react-native#initializing).
+
+Example:
+
+iOS
+
+```objectivec
+@import AEPCore;
+@import AEPLifecycle;
+@import AEPCampaignClassic;
+
+@implementation AppDelegate
+-(BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+  [AEPMobileCore setLogLevel: AEPLogLevelTrace];
+  [AEPMobileCore registerExtensions: @[ AEPCampaignClassic.class] completion:^{
+    [AEPMobileCore configureWithAppId:@"yourAppID"];
+    [AEPMobileCore lifecycleStart:@{@"contextDataKey": @"contextDataVal"}];
+  }
+  ];
+  return YES;
+}
+@end
+```
+
+Android
+
+```java
+import com.adobe.marketing.mobile.AdobeCallback;
+import com.adobe.marketing.mobile.InvalidInitException;
+import com.adobe.marketing.mobile.Lifecycle;
+import com.adobe.marketing.mobile.LoggingMode;
+import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.campaignclassic.CampaignClassic;
+
+...
+import android.app.Application;
+...
+public class MainApplication extends Application implements ReactApplication {
+  ...
+  @Override
+  public void on Create(){
+    super.onCreate();
+    ...
+    MobileCore.setApplication(this);
+    MobileCore.setLogLevel(LoggingMode.DEBUG);
+    try {
+      CampaignClassic.registerExtension();
+      MobileCore.configureWithAppID("yourAppID");
+      MobileCore.start(new AdobeCallback() {
+        @Override
+        public void call(Object o) {
+          MobileCore.lifecycleStart(null);
+        }
+      });
+    } catch (InvalidInitException e) {
+      ...
+    }
+  }
+}
+```
+
+### Importing the extension:
+
+```typescript
+import { CampaignClassic } from '@adobe/react-native-AEPCampaignClassic';
+```
+
+## API reference
+
+### Getting the SDK version:
+
+**Syntax**
+
+```typescript
+extensionVersion(): Promise<string>
+```
+
+**Example**
+
+```typescript
+const version = await CampaignClassic.extensionVersion();
+console.log(`AdobeExperienceSDK: Campaign Classic version: ${version}`);
+```
+
+### Register Device with Campaign Classic:
+
+This API dispatches a Campaign request content event containing registration information (device token, user key, and additional parameters).
+**Syntax**
+
+```typescript
+registerDeviceWithToken(
+    token: string,
+    userKey: string,
+    additionalParameters?: Record<string, any>
+): void
+```
+
+**Example**
+
+```typescript
+CampaignClassic.registerDeviceWithToken('myToken', 'myUserKey')
+);
+```
+
+### Tracking Notification Click:
+
+Dispatch event containing tracking notification from notification click.
+
+**Syntax**
+
+```typescript
+trackNotificationClickWithUserInfo(userInfo: Record<string, any>): void
+```
+
+**Example**
+
+```typescript
+CampaignClassic.trackNotificationClickWithUserInfo({ _mId: 'testId' });
```

Comment:
both `_mId` and `_dId` are required for the tracking request to be send. Let's update the examples to include both